### DISCUSSION TITLE
request is invalid on Discord

### DISCUSSION THREAD

**Discussion by Boby900** [nextauthjs/next-auth#11250] [Help] [answered]
Hii all,
I am trying to join the DIscord server but it's showing that **Invite Invalid**.
![Screenshot (565)](https://github.com/nextauthjs/next-auth/assets/111265239/4effc0ce-60ab-4305-9565-187ec98ddbd4)

**Comment by k-taro56** [comment, marked as answer]
Hi,

Thank you for your cooperation.

I have created a new invitation link, please use this it.
https://discord.gg/V7X2RRxrKS

Thank you.

**Comment by Boby900** [comment]
hey @k-taro56 can I open up a PR correcting the invitation Link given in the ReadMe for joining the Discord replacing old one with this one which you have provided.
Would that be ok?

**Reply by k-taro56** [reply]
I would say "Of course!", but this invite link redirects from https://discord.authjs.dev/, so changing the README is not a fundamental solution.
I have already contacted the server admin about this.

Thanks.

**Reply by Boby900** [reply]
okay, then.

**Comment by ndom91** [comment]
Hey folks, the [discord.authjs.dev](https://discord.authjs.dev) link has been updated :pray:
